src/internal/connector/sharepoint/data_collections.go

```diff
@@ -232,6 +243,7 @@ func collectLibraries(
 func collectPages(
 	ctx context.Context,
 	creds account.M365Config,
+	ac api.Client,
 	serv graph.Servicer,
```

Comment:
nit: gs instead of serv

Reply:
that value will soon disappear.  once the client is fully threaded through, we shouldn't pass around graph.Servicer interfaces.